manta-crypto/src/merkle_tree/inner_tree.rs

```diff
@@ -234,6 +258,11 @@ where
         self.set(rhs_index, rhs_digest);
         digest
     }
+
+    /// Removes the inner digest stored at 'index'
+    fn remove(&mut self, _index: usize) -> Bool {
+        false
+    }
```

Comment:
Make sure when designing the interfaces to preserve the boundaries, keep the variables the exact names like `index` not `_index`, this reflects in all the documentation and becomes the canonical name for that argument.

```suggestion
    /// Removes the inner digest stored at 'index'.
    ///
    /// # Implementation Note
    ///
    /// By default, this method does nothing and returns `false`. Implementations of this method may
    /// fail arbitrarily, and should only successfully remove a node if the implementation is
    /// efficient enough. Space and time tradeoffs should be studied to determine the usefulness of
    /// this method.
    #[inline]
    fn remove(&mut self, index: usize) -> bool {
        let _ = index;
        false
    }
```